flight/targets/sparky/fw/Makefile

```diff
@@ -41,24 +41,18 @@ MODULES += FirmwareIAP
 MODULES += Telemetry
 
 OPTMODULES += GPS
-OPTMODULES += CameraStab
 OPTMODULES += Autotune
 OPTMODULES += TxPID
-OPTMODULES += PathPlanner
-OPTMODULES += VtolPathFollower
-OPTMODULES += FixedWingPathFollower
```

Comment:
It would be a real shame to make Sparky not usable for nav and fixed wing. It's a nice option for fixed wing. I think we should drop HoTT, FrSky sensors, Crossfire telem, storm32 etc. before these.